docs/testing-guide.md

```diff
@@ -110,6 +110,20 @@ Unlike the installer ISO, there isn't anything specific that needs to be done wh
 a raw image. EIB can be used to overwrite the raw image attached to a VM and, when the VM is booted, it will use
 the newly built image.
 
+## Testing using virt-install (CLI)
+
+If you want to use a command line example to deploy the VM and test it with your EIB image generated, you could use the following commands:
+
+- First, we need to create a disk empty to be used by the VM:
+
+`qemu-img create -f qcow2 example.img 6G`
+
+- Then, we could use virt-install to create and define a VM using the EIB output image generated:
```

Comment:
Following on the previous comment: "Next, use `virt-install` to define and run a VM using the EIB generated image:"